README.md

```diff
@@ -45,15 +46,33 @@ The core code of the project lays inside [flutter/lib](flutter/lib) directory an
 This project is license under a [GPL-3.0](https://www.gnu.org/licenses/gpl-3.0.html) license.
 
 
-## Disclaimer & Credits
+## Credits
 
-* This game is a **fan-made**, **non-commercial** recreation of Djambi,
-  a public-domain board game designed by *Jean Anesto* in 1975.
-* This project is **not officially affiliated** with *Jean Anesto*, his heirs,
-  or any company that may have published Djambi historically.
-* All **original game rules** are used under the principle that game mechanics are not copyrightable.
-  However, any **original artwork, branding, or terminology** from the 1975 version remains the property of its
-  respective rights holders.
-* This adaptation is open-source and free to use/distribute for non-commercial purposes.
-* Images of [pieces](./flutter/assets/classic) are based on Djambi classic theme create by [Rsalen](https://commons.wikimedia.org/wiki/User:Rsalen).
-* If you represent the rights to *Djambi* and have concerns, please contact [me](mailto:hello@datonomi.com) for respectful resolution.
+Images of [pieces](./flutter/assets/classic) are based on Djambi classic theme create by [Rsalen](https://commons.wikimedia.org/wiki/User:Rsalen).
+
+## **Play Djambi Online** 
+  
+Experience Djambi directly in your web browser without any downloads:
+🔗 [Play Djambi](https://mabdelaal86.github.io/djambi/)
+
+## Download the Game
+
+You can download **Djambi** on your smartphone using the following links:
+
+- **Android (Google Play Store):**  
+  [![Get it on Google Play](https://img.shields.io/badge/Google%20Play-Download-blue?logo=google-play)](https://play.google.com/store/apps/details?id=com.datonomi.djambi)  
+
+<!-- - **Android (APKPure)**  
+  [![APKPure](https://img.shields.io/badge/APKPure-Download-blue)](https://apps.apple.com/app/id1234567890)  -->
+
+<!-- - **iOS (App Store):**  
+  [![Download on the App Store](https://img.shields.io/badge/App%20Store-Download-blue?logo=apple)](https://apps.apple.com/app/id1234567890)   -->
+
+## Contact Us
+
+**Questions? We're happy to help.**  
+**Want to contribute? We'd love to have you!**  
+
+**Email:** [hello@datonomi.com](mailto:hello@datonomi.com)  
+
+[**Privacy Policy**](https://datonomi.github.io/djambi/privacy-policy)
```

Comment:
I'm not sure if we need the privacy policy here in GitHub as it is only necessary for Google play store and it may be different from a store to another.